packages/image_picker/image_picker_ios/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
-## NEXT
+## 0.8.5+6
 
+* Updated description
```

Comment:
This needs to follow [the style guide](https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog-style), so it should be "Updates", and needs a period at the end.